src/node/rpc/test/frontend_test.cpp

```diff
@@ -522,7 +522,7 @@ TEST_CASE("process_bft")
     enclave::InvalidSessionId, user_caller_der);
   auto ctx = enclave::make_rpc_context(session, request.raw);
   ctx->execute_on_node = true;
-  frontend.process_bft(ctx);
+  frontend.process_bft(ctx, 1, 1);
```

Comment:
```suggestion
  constexpr auto prescribed_commit_version = 1;
  constexpr auto max_conflict_version = 1;
  frontend.process_bft(ctx, prescribed_commit_version, max_conflict_version);
```

And just to confirm - these are unrelated to the `{1, 1}` used to construct `rid` above?

Reply:
yes the two are not related